src/viewProviders/baseModels/parentedBase.ts

```diff
@@ -34,7 +34,7 @@ export abstract class ParentedBaseViewProvider<
    * - Topics view: `KafkaCluster`
    * - Schemas view: `SchemaRegistry`
    * - Flink Statements view: `FlinkComputePool`
-   * - Flink Artifacts view: `FlinkComputePool`
+   * - Flink Databases view: `KafkaCluster` (as Flink database)
```

Comment:
Unrelated change, but bothered me when I highlighted over the `.resource` from somewhere else and saw the old view reference